pkgs/desktops/xfce/default.nix

```diff
@@ -169,8 +169,6 @@ lib.makeScope pkgs.newScope (self: with self; {
   xfce4_power_manager = xfce4-power-manager;
   xfce4_appfinder = xfce4-appfinder;
   xfce4_dev_tools = xfce4-dev-tools;
-  xfce4mixer = xfce4-mixer;
-  xfce4mixer_pulse = xfce4-mixer-pulse;
```

Comment:
these (and more) don't throw, did you mean to remove those here as well?

Reply:
Yes, they did throw since they pointed to a throw alias

Reply:
Check lines 207 and 220 of the same file.